### PR TITLE
[53] 404 Page

### PR DESCRIPTION
**Pull request by yijo2004**
Updated 404 Page to match Figma on desktop page.  Links are not linked to the proper pages and mobile page not done.

### CODE REVIEW

**Review by yijo2004 — COMMENTED** [review]
Updated to work with mobile, links are not linked yet due to the pages not being finished.

File: frontend/src/pages/404.js [hunk 1]
```diff
@@ -2,11 +2,40 @@ import * as React from "react";
...
+          <div className="font-bold text-xl lg:text-3.5xl md:text-3.5xl sm:text-3.5xl leading-tighter text-Shades-100 uppercase pb-20">WE COULDN'T FIND THAT PAGE</div>
+          <div className="font-medium pb-6">Try going back or clicking one of the links below:</div>
+          <div className="pb-3">
+            <a key={`browseLink-sundayCelebrationInfo`} href={"/"}>
+              <link1>Sunday Celebration Info</link1>
+            </a>
+          </div>
+          <div className="pb-3">
+            <a key={`browseLink-lifegroupSignup`} href={"/"}>
+              <link1>Life Group Sign Up</link1>
```
Comment (thedomcab) [inline frontend/src/pages/404.js:28]: link1 is non-standard html element. what are you. if you want to use an inline html element, you can use span. the same goes for the one Below. Also use gatsby Link element for routing to to different pages on the site. you need to important it. Thanks.

File: frontend/src/pages/404.js [hunk 2]
```diff
@@ -2,11 +2,40 @@ import * as React from "react";
```
Comment (thedomcab) [inline frontend/src/pages/404.js:2]: for responsive if all the breakpoints are the same, you just need to use the smallest breakpoint  for example `lg:pb-20 md:pb-20 sm:pb-20 mb-80` should be this: `sm:pb-20 mb-80`

**Comment by yijo2004** [comment]
Resolved issues, waiting for review

File: frontend/src/pages/404.js [hunk 3]
```diff
@@ -1,12 +1,41 @@
...
-    <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
+    <div className={container}>
+      <div className="justify-center flex pb-4 sm:pb-20 mb-80 sm:flex-row flex-col">
+        <div className="basis-1/3 pt-10 pb-0 sm:pb-10 order-1 sm:mr-20">
+          <h1 className="relative text-left px-8">404: Error</h1>
+          <img className="object-contain w-full sm:px-8 px-24 sm:pt-4" src={ErrorImage} alt="404 Error" />
+        </div>
+        <div className="pl-2.5 px-8 pt-4 lg:pt-12 order-2 mt-0 sm:mt-20 mx-14 text-center sm:text-left">
+          <div className="xl:font-extrabold font-bold text-4xl sm:text-8xl sm:tracking-widest leading-tighter">OOPS!</div>
+          <div className="font-bold text-xl sm:text-3.5xl leading-tighter text-Shades-100 uppercase pb-20">WE COULDN'T FIND THAT PAGE</div>
```
Comment (thedomcab) [inline frontend/src/pages/404.js:19]: I found a minor linting issue. Fix this code should be all good. thanks. much appreciated.
```suggestion
          <div className="font-bold text-xl sm:text-3.5xl leading-tighter text-Shades-100 uppercase pb-20">WE COULDN&apos;T FIND THAT PAGE</div>
```

**Review by thedomcab — APPROVED** [review]

**Pull request merged**